clients/privacy-center/components/ConsentItemCard.tsx

```diff
@@ -83,11 +80,10 @@ const ConsentItemCard: React.FC<ConsentItemProps> = ({
           </Link>
         </Box>
         <RadioGroup
+          value={value ? "true" : "false"}
```

Comment:
The use of strings in this file is throwing me off, but it looks like the `RadioGroup` `value` only takes a string. 🤷‍♀️